src/api/routes/faucets.ts

```diff
@@ -88,7 +88,7 @@ export function createFaucetRouter(db: DataStore): RouterWithAsync {
         return;
       }
 
-      const stxAmount = 3_333_333_333_333_333; // Minimum required for stacking
+      const stxAmount = 3_000_000_000_000; // Minimum required for stacking
```

Comment:
@lgalabru once you include fees to make the stacking request, this won't be enough... So everyone needs to make at least 2 requests

Reply:
never mind, it looks like the amount is 2_000_000_000_000, so plenty for fees...